src/mitol/common/envs.py

```diff
@@ -220,6 +220,54 @@ def parse_delimited_list(name, value, default):
     return [item.strip(" ") for item in parsed_value]
 
 
+CRONTAB_KEYS = ["minute", "hour", "day_of_week", "day_of_month", "month_of_year"]
+
+
+def parse_crontab_kwargs(name, value, default):
+    """
+    Parses a crontab string into structured kwargs
+
+    Argumments:
+        value (str):
+            the crontab string value
+        default (str or dict):
+            the default crontab as either a string or dict
+
+    Returns:
+        dict:
+            the parsed crontab kwargs
+    """
+
+    if isinstance(value, str):
+        units = value.split(" ")
+
+        if len(units) != len(CRONTAB_KEYS):
```

Comment:
Since we can create the crontab without defining all the values at least for now, with this change, it would be required to add value for all the units defined here.
 
One might wonder if the parser could set default `*` for any missing unit if not provided to the parser, In my opinion, it's not an issue, at least it makes things standardized everywhere.

Reply:
I feel like that could cause an ambiguity because the code would be making an assumption about an incomplete value and it can't distinguish between a purposeful omission and an accidental one so I think it's better to require all 5 values if you're passing a `str`. The defaults are all `*` if on `crontab()` so if you'd be wanting to just pass one or two values using the `dict` type is best.

Reply:
This makes sense.